resources/build/publish-minimum-versions.sh

```diff
@@ -37,5 +37,30 @@ function generate_table {
   echo "> ### This file is auto-generated by publish-minimum-versions.sh. Do not modify"
 }
 
-cp "$KEYMAN_ROOT/docs/minimum-versions.md.in" "$KEYMAN_ROOT/docs/minimum-versions.md"
-generate_table >> "$KEYMAN_ROOT/docs/minimum-versions.md"
+do_build() {
+  cp "$KEYMAN_ROOT/docs/minimum-versions.md.in" "$KEYMAN_ROOT/docs/minimum-versions.md"
+  generate_table >> "$KEYMAN_ROOT/docs/minimum-versions.md"
+}
+
+do_test() {
+  # Verify that minimum-versions.md matches the generated data
+
+  local new_table="$(mktemp)"
+  cp "$KEYMAN_ROOT/docs/minimum-versions.md.in" "$new_table"
+  generate_table >> "$new_table"
+  diff --suppress-common-lines -yw "$KEYMAN_ROOT/docs/minimum-versions.md" "$new_table" || \
```

Comment:
Thoughts on just regenerating the file with do_build()
and then having `git diff -w ...` flag if the file changed?

Reply:
I wanted to avoid git-fu with the build agents.